### DISCUSSION TITLE
Start code server as non root user

### DISCUSSION THREAD

**Discussion by Blacksquadman1** [coder/code-server#6765] [Q&A] [answered]
good day first 
i have an ubuntu 22.04 server i have installed the code server there but when i start the code server with a non root user i get the following error message.
` listen EACCES: permission denied IP:PORT`
i have no idea what authorization the user needs to start the program

**Comment by code-asher** [comment, marked as answer]
What port are you listening on?  In Linux all ports below 1024 are privileged.

**Reply by Blacksquadman1** [reply]
ah ok thanks good i did not know about the ports i had it on 600 now it is on port 1025 now it works thanks for the help
